.babelrc

```diff
@@ -6,6 +6,7 @@
   ],
   "presets": ["es2015", "stage-2"],
   "plugins": [
+    "jsx-control-statements",
```

Comment:
Right now we're mostly doing it like `<div>{!!(condition) && <div />}</div>` (abusing short-circuit evaluation, since React will never output `false` in the HTML) which is kinda gross and annoying to indent. We'll have to convert those later.

I guess it made more sense to use an uppercase later since it's closer in spirit to a custom component than a native element. Except it's not a component at all, so dunno. :)